tensorflow/lite/kernels/depthwise_conv.cc

```diff
@@ -398,6 +406,35 @@ TfLiteStatus EvalQuantizedPerChannel(TfLiteContext* context, TfLiteNode* node,
   return kTfLiteOk;
 }
 
+TfLiteStatus EvalQuantizedPerChannel16x8(
+    TfLiteContext* context, TfLiteNode* node, TfLiteDepthwiseConvParams* params,
+    OpData* data, const TfLiteTensor* input, const TfLiteTensor* filter,
+    const TfLiteTensor* bias, TfLiteTensor* output) {
+  DepthwiseParams op_params;
+  op_params.padding_type = PaddingType::kSame;
```

Comment:
this should read from params right? if valid padding is not supported for now, please leave a TODO

Reply:
I'm not sure what's the best way to leave a TODO for OSS, +jdduke

Reply:
I see that in all other functions this parameter is set the same way, so I removed this TODO.
Please let me know what is the best way to go forward with this PaddingType.

Reply:
@renjie-liu please file an internal bug if you think we ought to be handling other padding types. As noted, this is matching existing behavior.

Reply:
Sounds good, please add:

TODO(b/150034149): Support Both Same & Valid padding type.